Readme.md

```diff
@@ -53,12 +53,12 @@ Stack implements ```Interop\Http\ServerMiddleware\DelegateInterface```.
 // you decide what middleware you want to put in a stack.
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
-use Interop\Http\ServerMiddleware\DelegateInterface;
-use Interop\Http\ServerMiddleware\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Psr\Http\Server\MiddlewareInterface;
 
 class TrickyMiddleware implements MiddlewareInterface
 {
-    public function process(ServerRequestInterface $request, DelegateInterface $frame) : ResponseInterface
+    public function process(ServerRequestInterface $request, RequestHandlerInterface $frame) : ResponseInterface
```

Comment:
why not name the `$frame` to `$handler` ? BC break ?

Reply:
thank you for the hint - missed that one.